docs/config.md

```diff
@@ -55,11 +57,23 @@ Neurobagel offers different deployment profiles that allow you to spin up specif
         If you choose to use the `local_federation` profile, 
         you will have to [manually configure your `local_nb_nodes.json` file](#local_nb_nodesjson).
 
-You can then launch these profiles by using the `--profile` flag with `docker compose`, e.g.:
+#### Launching a profile
+You can then launch a specific profile in one of two ways:
 
-```bash
-docker compose --profile full_stack up -d
-```
+- Using the `--profile` flag with `docker compose`, e.g.:
+    ```bash
+    docker compose --profile full_stack up -d
+    ```
+- Editing the value of `COMPOSE_PROFILES` (a [built-in Docker Compose environment variable](https://docs.docker.com/compose/how-tos/environment-variables/envvars/#compose_profiles)) in `.env`, which defines the 
+default profile that should be started when no profile is specified:
+  ``` sh title=".env"
+  ...
+  COMPOSE_PROFILES=full_stack
+  ```
+  Then, you can launch the profile you have defined in `.env` simply using:
+  ```bash
+  docker compose up -d
+  ```
```

Comment:
I see your point - agree to remove this documentation for now. To clarify though, I think changing the default value in .env _may_ actually be more error-proof & reproducible, since you can essentially change it in one place & do `docker compose up -d`, and then just look at the `.env` and know what profile your recipe was ran with last time.

This is actually how we have the .env files set up in the nodes that we host, since we have some that require `local_federation` and some that are `local_node` or `full_stack` - I just change it in .env so we can do `docker compose up -d` directly. Otherwise, you might forget which profile a specific .env/recipe configuration is meant to correspond to, thus requiring you documenting your `docker compose` command separately. With the current approach, as we've seen, I think it's more likely that users accidentally do `docker compose up -d` and sneakily launch the profile we set with `COMPOSE_PROFILES` in .env.

That said, given that we've changed the default now to what I suppose most users will actually be using, I don't think this is as big of an issue. Can revisit in the future if it comes up!